stories/primer/alpha/dialog_stories.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require "primer/alpha/dialog"
+
+class Primer::Alpha::DialogStories < ViewComponent::Storybook::Stories
+  layout "storybook_preview"
+
+  story(:dialog) do
+    controls do
+      title "github"
+      description ""
+      select(:tag, Primer::Alpha::Dialog::TAG_OPTIONS, Primer::Alpha::Dialog::DEFAULT_TAG)
+      href "#"
+    end
+
+    content do |c|
+
+    end
+  end
+end
```

Comment:
Generally trying to stop writing stories and start writing previews `tests/previews/` you can run script/lookbook to see the dialog and test it